{"uid":"c9547659d424a1a7","name":"Валидация поля description: None","fullName":"tests.test_backend.board.board_validation_test#test_create_board_group_description_validation","historyId":"8224906e80748855ee2ae973ed2a412d","time":{"start":1749744321676,"stop":1749744322002,"duration":326},"status":"passed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":true,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_verify_url","time":{"start":1749744310971,"stop":1749744310971,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"base_url","time":{"start":1749744310971,"stop":1749744310971,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"pytestconfig","time":{"start":1749744310972,"stop":1749744310972,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"delete_output_dir","time":{"start":1749744310972,"stop":1749744310972,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"owner_client","time":{"start":1749744310972,"stop":1749744311581,"duration":609},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"temp_space","time":{"start":1749744311581,"stop":1749744312659,"duration":1078},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"temp_project","time":{"start":1749744312659,"stop":1749744316443,"duration":3784},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"temp_board","time":{"start":1749744319586,"stop":1749744319951,"duration":365},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"testStage":{"status":"passed","steps":[{"name":"Проверка, что API вернул 200 при None (optional)","time":{"start":1749744322002,"stop":1749744322002,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"attachments":[{"uid":"398d77bc5734b870","name":"stdout","source":"398d77bc5734b870.txt","type":"text/plain","size":69}],"parameters":[],"shouldDisplayMessage":false,"stepsCount":1,"attachmentsCount":1,"hasContent":true},"afterStages":[{"name":"temp_space::0","time":{"start":1749744347725,"stop":1749744348325,"duration":600},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"labels":[{"name":"tag","value":"backend"},{"name":"parentSuite","value":"tests.test_backend.board"},{"name":"suite","value":"board_validation_test"},{"name":"host","value":"pkrvmxyh4eaekms"},{"name":"thread","value":"2546-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_backend.board.board_validation_test"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"description","value":"None"},{"name":"expected_status","value":"200"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"history":{"statistic":{"failed":0,"broken":7,"skipped":0,"passed":2,"unknown":0,"total":9},"items":[{"uid":"533a5195db0936af","reportUrl":"https://vaizcom.github.io/autotests/123//#testresult/533a5195db0936af","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f6ed7455e50>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749744003433,"stop":1749744003433,"duration":0}},{"uid":"1d0998b19deccc37","reportUrl":"https://vaizcom.github.io/autotests/122//#testresult/1d0998b19deccc37","status":"passed","time":{"start":1749743625723,"stop":1749743625978,"duration":255}},{"uid":"85ee0b2e9cf0d3f9","reportUrl":"https://vaizcom.github.io/autotests/119//#testresult/85ee0b2e9cf0d3f9","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fc7b5913490>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749742820572,"stop":1749742820572,"duration":0}},{"uid":"dbc6560a97d5cfe","reportUrl":"https://vaizcom.github.io/autotests/118//#testresult/dbc6560a97d5cfe","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f9cb88f1f10>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749742667966,"stop":1749742667966,"duration":0}},{"uid":"2bdaf69f1ebab8","reportUrl":"https://vaizcom.github.io/autotests/117//#testresult/2bdaf69f1ebab8","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f2b9f21f250>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749742497748,"stop":1749742497748,"duration":0}},{"uid":"e499a495825420b6","reportUrl":"https://vaizcom.github.io/autotests/116//#testresult/e499a495825420b6","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7efcc7f78ad0>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749741234173,"stop":1749741234173,"duration":0}},{"uid":"f51a1ddea584fdec","reportUrl":"https://vaizcom.github.io/autotests/115//#testresult/f51a1ddea584fdec","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f28d1ddcd10>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749740826694,"stop":1749740826694,"duration":0}},{"uid":"53c92d53627baf08","reportUrl":"https://vaizcom.github.io/autotests/114//#testresult/53c92d53627baf08","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f271b110ad0>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749740653435,"stop":1749740653435,"duration":0}}]},"tags":["backend"]},"source":"c9547659d424a1a7.json","parameterValues":["None","200"]}